.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "cSpell.words": [
+    "Popconfirm"
```

Comment:
Make sure to remove this. I think you need to add this file to your global gitignore